src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/VsTestConsoleWrapper.cs

```diff
@@ -3,6 +3,7 @@
 
 namespace Microsoft.TestPlatform.VsTestConsole.TranslationLayer
 {
+    using System;
```

Comment:
I know this is how it's been for a long time, but is there anything that would prevent us from not having to spawn vstest.console, but rather start all classes in the same process as VsTestConsoleWrapper?

Reply:
I'm not sure I fully get the question. Do you mean to remove the abstractions offered by `vstest.console` being a separate process and then do everything in the same process that runs the `VsTestConsoleWrapper`, in this case TE ? This would mean a major architectural overhaul of TP, not to mention the compatibility challenges posed by such a change, because not everyone uses the Translation Layer.

AFAIK, Rider maintains JSON compatibility with the messages passed between the Translation Layer and `vstest.console` but they do not use the actual component. Another case would be the .NET SDK which doesn't use the Translation Layer, but it uses `vstest.console` through some MSBuild targets.